app/models/relationship.rb

```diff
@@ -0,0 +1,6 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: User.name
+  belongs_to :followed, class_name: User.name
+  validates :follower_id, presence: true
```

Comment:
Viết theo cách của anh thì tốt hơn so với validates :follower_id . Bởi vì
+validates :follower_id : sẽ không kiểm tra xem user đó có tồn tại hay không mà sẽ lưu thẳng vào csdl
+validates :follower : sẽ an toàn hơn khi kiểm tra sự tồn tại của user, follower_id có nill hay không